extract_from_hg.py

```diff
@@ -195,15 +196,105 @@ def commitCset(cset):
     runCommand(['git', 'commit', '-m', commitMsg, '--author=' + cset.user(),
                 '--date=' + str(csetDate)])
 
+def formatCommitMessageLine(line):
+    # Now do translations on the line.
+
+    # First, strip off any review flags from the end of line.
+    line = re.sub(r" *\[*rs*\=.*", "", line)
```

Comment:
Please do this as a multi-line string with comments  (like the localeList stuff earlier in the script) to make the regexp itself more readable.